lib/response_bank/middleware.rb

```diff
@@ -13,7 +13,7 @@ def initialize(app)
 
     def call(env)
       env['cacheable.cache'] = false
-      gzip = env['gzip'] = env['HTTP_ACCEPT_ENCODING'].to_s.include?("gzip")
+      env['br'] = env['HTTP_ACCEPT_ENCODING'].to_s.include?("br")
```

Comment:
you need to support both br and gzip from the request coming in. Functionally only be one used (passed in from cloudflare), but we need to support both `Accept-Encoding: gzip` or `Accept-Encoding: br`.